pyproject.toml

```diff
@@ -11,6 +11,8 @@ python-dotenv = "^0.14.0"
 requests = "^2.31.0"
 pytest = "^8.1.1"
 black = "^24.2.0"
+pymongo = "^4.10.1"
+mongomock = "^4.2.0.post1"
```

Comment:
What an odd version string! Not important, but I [learned something looking this up](https://peps.python.org/pep-0440/#post-releases)
> Some projects use post-releases to address minor errors in a final release that do not affect the distributed software (for example, correcting an error in the release notes).